### ISSUE TITLE
SolrNet Priorities

### ISSUE THREAD

**Issue by xavier-morera** [SolrNet/SolrNet#270]
Hi, here is a form to collect information on SolrNet priorities to decide what can be taken first. Please share with other SolrNet users. 

https://goo.gl/forms/W7eYFcpAhGvvbURS2

From this feedback we can then talk and decide what path should we take

**Comment by stanuku** [comment]
@xavier-morera: Hi, the form doesn't let you to multiple items and prioritize them. Just wondering if you could change it such that we can assign a rating/ranking lets say from 1 to 5. (5 being most important)

For example:

4 - SolrCloud
2 - Support for newer versions of Solr, including breaking changes
3 - AsyncOperations
1 - Support for .NET Core
Other: 5 - Upgrade Ninject

**Comment by xavier-morera** [comment]
Updated for multiple choice, including "Other"

**Comment by xavier-morera** [comment]
Here is an update

![image](https://cloud.githubusercontent.com/assets/7174278/24716307/df7cf70e-19eb-11e7-86ec-155170024581.png)

![image](https://cloud.githubusercontent.com/assets/7174278/24716322/ed0ac8c4-19eb-11e7-8ee9-f4bb4bf5c85a.png)

I will let it run for one more week and will post the final results on April 17

**Comment by gjunge** [comment]
Hi @xavier-morera , any more entries?

**Comment by xavier-morera** [comment]
Here is the final one. I do have a problem though which is that I am still missing a write permission from @mausch, so I am still waiting for this. 

![image](https://cloud.githubusercontent.com/assets/7174278/25137606/8916b59c-2415-11e7-83a0-c8aaeb3cfc17.png)
![image](https://cloud.githubusercontent.com/assets/7174278/25137634/97433de8-2415-11e7-86d2-ff13c3d45dc1.png)

**Comment by xavier-morera** [comment]
I got the permission and accepted a pull request for a documentation change. It is time to start working. I will review some of the outstanding pull requests to see where we stand and bring some of them for review. I believe Gitflow workflow is a good branching strategy to manage development and versioning. Thoughts?

**Comment by gjunge** [comment]
That's great news!
I agree with you on gitflow. The strategy/instructions should be included in the how to contribute section of the [readme.md](https://github.com/SolrNet/SolrNet/blob/master/Documentation/README.md).

**Comment by stanuku** [comment]
@xavier-morera: I'vent used Gitflow workflow, I tried to get familiar with it after your post and came acrosss this http://endoflineblog.com/gitflow-considered-harmful. I'm sure there are arguments 'for' it too. Just thought you may be interested to see the author's opinion/experience 'against' it.

**Comment by xavier-morera** [comment]
Thanks @stanuku for providing the link. I will read it, digest it and comment soon. 

Regarding Gitflow, I've used it and IMHO it works well. I did have some struggles with it at the beginning, but when I started to use a GUI (namely SourceTree, although I use Github desktop too) it started to become a lot easier for me. I even took the time on creating a course on how to use Git with a GUI at Pluralsight to help other people that struggle with Git when getting started ([here is the link](https://www.pluralsight.com/courses/using-git-with-gui) in case you are curious)
